keras_cv/models/stable_diffusion/text_encoder.py

```diff
@@ -49,16 +69,18 @@ def call(self, inputs):
         positions = self.position_embedding(positions)
         return tokens + positions
 
+def quick_gelu(x):
+    return x * tf.sigmoid(x * 1.702)
 
 class CLIPEncoderLayer(keras.layers.Layer):
-    def __init__(self, **kwargs):
+    def __init__(self, embed_dim, num_heads, use_quick_gelu=True, **kwargs):
```

Comment:
instead of `use_quick_gelu`, can we pass in an activation function, and defaults to None?

Reply:
Updated. Can you see if it is ok now?